tsdb/shard.go

```diff
@@ -243,6 +243,13 @@ func (s *Shard) WritePoints(points []Point) error {
 			s.mu.RLock()
 			mf := s.measurementFields[p.Name()]
 			s.mu.RUnlock()
+
+			// If a measurement is dropped while writes for it are in progress, this could be nil
+			// See #2608
```

Comment:
Minor: I haven't seen us do this before i.e. reference bug numbers. I think the comment is pretty good as it is.